coralproject/talk
# [CORL-748] Update admin login to match new branding styles

## What does this PR do?

Updates styles on the admin login page to match the new branding changes.

## How do I test this PR?

- Head to admin login when signed out
- See login page when signed out is changed
- Head to admin login with commenter user
- See appropriate permission denied page for commenter user
